cdk/lib/frontend.ts

```diff
@@ -1,19 +1,17 @@
-import { join } from "path";
 import {
   ComparisonOperator,
   Metric,
   TreatMissingData,
 } from "@aws-cdk/aws-cloudwatch";
 import { InstanceClass, InstanceSize, InstanceType } from "@aws-cdk/aws-ec2";
 import { FilterPattern, LogGroup, MetricFilter } from "@aws-cdk/aws-logs";
-import { CfnInclude } from "@aws-cdk/cloudformation-include";
```

Comment:
You should be able to remove `@aws-cdk/cloudformation-include` from your `package.json` too. We recently discovered that this pulls in _a lot_ of transitive dependencies and significantly slows down builds (https://github.com/guardian/security-hq/pull/435), so it's definitely worth removing it if you can.

Reply:
Oh that's a great spot - thanks! I'll remove that dep now

Reply:
Yep that's given us lots of red in the `yarn.lock`!